### ISSUE TITLE
 The \hat and \dagger operator are orribly rendered

### ISSUE THREAD

**Issue by mesonepigreco** [arXiv/html_feedback#622]
### Description

The \hat a^\dagger, operators for creation and annihilation, have a really poor design and readability in the HTML version. In particular because it is a double superscript, while the dagger operator should be on the same height independently on the height of the letter (like a or b). , it

### (Optional:) Please add any files, screenshots, or other information here.

Example:
https://arxiv.org/html/2402.15664v1

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

06ebb33c-8322-4e58-9991-84fe27ba61f2

### Paper URL

https://arxiv.org/html/2402.15664v1

### Browser

Chrome/121.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
I am attaching a screenshot of today's MathML rendering in Chrome here for reference:

<img width="213" height="55" alt="Image" src="https://github.com/user-attachments/assets/34d06691-f088-4e77-86b1-e55d1d3c2af6" />

And cross-referencing a relevant MathML Core issue:
https://github.com/w3c/mathml-core/issues/301
